contributors/index.js

```diff
@@ -1,4 +1,13 @@
-const list = [{
+const list = [
+	{
```

Comment:
Please add your details to the bottom of the list!

Reply:
Sure, thanks.